server/topic.go

```diff
@@ -3320,6 +3334,57 @@ func (t *Topic) subsCount() int {
 	return len(t.perUser)
 }
 
+// Adds a new multiplex proxied session to the topic's clusterWriteLoop.
+func (t *Topic) addProxiedSession(s *Session) {
```

Comment:
reflect.Select can select from up to 2^16 channels. Is it possible to have more than 2^14 sessions?

Reply:
Or maybe these are multiplexing sessions, i.e. one per cluster member?

Reply:
The max number of multiplexing sessions we have per topic is `#number of nodes - 1`.
In practice, we should never go all the way up to `2^16`. Even 10-20 nodes is already a rather large footprint.